src/Framework/AspNetCoreAnalyzers/test/Mvc/DetectAmbiguousActionRoutesTest.cs

```diff
@@ -321,5 +321,103 @@ static void Main(string[] args)
         // Act & Assert
         await VerifyCS.VerifyAnalyzerAsync(source, expectedDiagnostics);
     }
+
+    [Fact]
+    public async Task DuplicateRoutes_HasHttpAttributes_NoDiagnostics()
+    {
+        // Arrange
+        var source = @"
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc;
+public class MyController : Controller
+{
+    [HttpGet]
+    [Route(""Person"")]
+    public IActionResult Get() => Ok(""You GET me"");
+    
+    [HttpGet(""PersonGet"")]
```

Comment:
```suggestion
```

Is this really necessary for the scenario being validated?

Reply:
It validates that the GET verb isn't applied to `[Route("Person")]` when `HttpGet` (or friends) has a route of its own.